app-template/values.schema.json

```diff
@@ -78,6 +78,11 @@
           "type": "boolean",
           "description": "Create a Service for your application"
         },
+        "type": {
+          "title": "Service Type",
+          "type": "string",
+          "description": "Specify the type of Service to create."
+        },
```

Comment:
Can you add the `enum` property with different Service types? That will make the type field a dropdown instead of a plain input field

Reply:
@petar-cvit done can you please review now

Reply:
![image](https://github.com/user-attachments/assets/1493d201-2ed8-4adc-92be-a244d7426547)